moonraker/app.py

```diff
@@ -347,6 +354,20 @@ async def _process_http_request(self, method):
         self.finish({'result': result})
 
 
+class LocalOctoRequestHandler(LocalRequestHandler):
+    async def _process_http_request(self, method):
```

Comment:
I recently refactored the dynamic request handlers to accept a JSON body to better support the database APIs, so it should no longer be necessary to add a new handler. Simply register endpoints with the protocol set to ["http"] and set the wrap_result argument to False. Your handler callbacks will need to accept a "web_request" object, with which you can use to parse the arguments or get the args dict directly.

Reply:
Ah, thanks. That will be useful. I wasn't too fond of this workaround.